com.unity.netcode.gameobjects/Tests/Runtime/DistributedAuthority/DistributedAuthorityCodecTests.cs

```diff
@@ -45,6 +53,13 @@ internal class DistributedAuthorityCodecTests : NetcodeIntegrationTest
         private static readonly ushort k_TransportPort = GetPortToBind();
         private const int k_ClientId = 0;
 
+        private bool m_EnsureVariableLengthSafety;
+
+        public DistributedAuthorityCodecTests(NetworkConfigOptions configOptions)
```

Comment:
Shouldn't this be:
` public DistributedAuthorityCodecTests(NetworkConfigOptions configOptions) : base()
`
?